src/main/java/com/faforever/api/data/domain/Faction.java

```diff
@@ -26,7 +23,6 @@
     this.string = string;
   }
 
-  @JsonCreator
```

Comment:
Why removed?

Reply:
well we said all string, so we want to send it as string and receive it as string

Reply:
Ok. Since we don't pass any replays it should be fine.